higlass_manage/update_viewconfs.py

```diff
@@ -0,0 +1,194 @@
+import sys
+import click
+import os.path as op
+import sqlite3
+import shutil
+import docker
+
+from .common import (
+    get_data_dir,
+    get_site_url,
+    get_port,
+    hg_name_to_container_name,
+    SQLITEDB,
+)
+
+from .stop import _stop
+
+
+@click.command()
+@click.option(
+    "--old-hg-name",
+    help="The name of the running higlass container that needs to be updated.",
+    required=False,
+)
+@click.option(
+    "--old-site-url",
+    help="site-url at the old location."
+    " Provide this when higlass container"
+    " one is updating is not running.",
```

Comment:
This phrasing is a little easier for me to parse:

```suggestion
    " to be updated is not running.",
```